src/ci/docker/scripts/musl-toolchain.sh

```diff
@@ -3,7 +3,7 @@
 #
 # Versions of the toolchain components are configurable in `musl-cross-make/Makefile` and
 # musl unlike GLIBC is forward compatible so upgrading it shouldn't break old distributions.
-# Right now we have: Binutils 2.27, GCC 6.3.0, musl 1.1.18
+# Right now we have: Binutils 2.27, GCC 6.3.0, musl 1.1.22
```

Comment:
This seems wrong. Looking at https://github.com/richfelker/musl-cross-make/blob/v0.9.7/Makefile, this still uses musl 1.1.18. Maybe you want to update to v0.9.8, or do this as another PR?